build/updatehistorictzdb.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -e
+
+rm -rf tmp
```

Comment:
If you're expecting this to be run from a particularly directory, perhaps add something like:

```sh
cd $(dirname $0)/..
```

before this.

Alternatively (or additionally) use something like

```sh
declare -r TEMPPATH=$(mktemp -d)
trap "rm -rf ${TEMPPATH}" EXIT
```

To give yourself a temporary directory to build into.